content/en/docs/reference/command-line-tools-reference/feature-gates-removed.md

```diff
@@ -156,6 +156,10 @@ In the following table:
 | `DisableAcceleratorUsageMetrics` | `false` | Alpha | 1.19 | 1.19 |
 | `DisableAcceleratorUsageMetrics` | `true` | Beta | 1.20 | 1.24 |
 | `DisableAcceleratorUsageMetrics` | `true` | GA | 1.25 | 1.27 |
+| `DownwardAPIHugePages` | `false` | Alpha | 1.20 | 1.20 |
+| `DownwardAPIHugePages` | `false` | Beta | 1.21 | 1.21 |
+| `DownwardAPIHugePages` | `true` | Beta | 1.22 | 1.26 |
+| `DownwardAPIHugePages` | `true` | GA | 1.27 | 1.29 |
```

Comment:
```suggestion
| `DownwardAPIHugePages` | `true` | GA | 1.27 | 1.28 |
```
it should be 1.28 not 1.29, as `DownwardAPIHugePages` will removed in k8s v1.29 so it is GA till k8s v1.28